display/pfobjects.py

```diff
@@ -116,8 +123,12 @@ def draw(self, projection, opt=''):
             
 class GStraightTrajectory(GTrajectory):
     def __init__(self, description):
-        super(GStraightTrajectory, self).__init__(description,
-                                                  linestyle=2, linecolor=1)
+        self.desc = description
+        if abs(self.desc.pdgid()) in [22]:
+          super(GStraightTrajectory, self).__init__(description,
+                                                  linestyle=2, linecolor=2)
+        elif abs(self.desc.pdgid()) in [12, 14, 16]:
```

Comment:
Le 21 juil. 2016 à 14:29, Mariiina <notifications@github.com> a écrit :

> In display/pfobjects.py:
> 
> > @@ -116,8 +123,12 @@ def draw(self, projection, opt=''):
> >              
> >  class GStraightTrajectory(GTrajectory):
> >      def __init__(self, description):
> > -        super(GStraightTrajectory, self).__init__(description,
> > -                                                  linestyle=2, linecolor=1)
> > +        self.desc = description
> > +        if abs(self.desc.pdgid()) in [22]:
> > +          super(GStraightTrajectory, self).__init__(description,
> > +                                                  linestyle=2, linecolor=2)
> > +        elif abs(self.desc.pdgid()) in [12, 14, 16]:
> I thought that we used simulated particles in the display that's why I added the neutrinos. Indeed, as we used the reconstructed particles, display the neutrinos is useless.
> 
> However, I can see on the displays some dot black lines. These lines reconstructed which kind of particles?
> 
> 

photons and stable neutral hadrons (Klong, neutrons, etc.) — not pi0s because pi0s promptly decay to photons.

> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>